docs/Getting-Started.md

```diff
@@ -176,15 +176,15 @@ async function routes (fastify, options) {
 
   fastify.get('/animals', async (request, reply) => {
     const result = await collection.find().toArray()
-    if (result.length === 0) {
+    if (result?.length) {
```

Comment:
Fastify supports node.js 10 and 12 that does not support this syntax by default so it would be better to keep the old check

Would you mind targeting the `next` branch instead?

Reply:
Thanks for your feedback, I updated target branch. Could I make another request for the same topic replacing
if(result?.length) with if(result && result.length) ?

Reply:
`collection.find().toArray()` must return `Array` even if no document exist. I think it do not need to change.

Reply:
Good, thank you. I rolled back this line.